charts/alluxio/templates/_helpers.tpl

```diff
@@ -258,7 +258,7 @@ resources:
           {{- else }}
         - name: {{ $mediumName }}
           emptyDir:
-            medium: {{ eq .mediumtype "MEM" | ternary "Memory" "" }}
+            medium: {{ eq .mediumtype "MEM" | ternary "Memory" (or (eq .mediumtype "SSD") (eq .mediumtype "HDD") | ternary "" (.mediumtype | quote)) }}
```

Comment:
Please update chart version and change log.

Reply:
Done. Thx.